docdb/docdb.go

```diff
@@ -125,11 +125,27 @@ func (d *DocDB) GetDocDB(ctx context.Context, name string) (*docdb.DBCluster, er
 		return nil, apierror.New(apierror.ErrInternalError, msg, nil)
 	}
 
-	log.Debugf("getting documentDB cluster and instance(s) with output: %+v", out)
+	log.Debugf("getting documentDB cluster and instance(s) output: %+v", out)
 
 	return out.DBClusters[0], err
 }
 
+// GetDocDBTags gets the tags for a documentDB cluster
+func (d *DocDB) GetDocDBTags(ctx context.Context, arn *string) ([]*docdb.Tag, error) {
```

Comment:
so the cluster doesn't return the tags???  lame...

Reply:
Apparently this was built by a completely different team 😆